heartbeat/scheduler/scheduler_test.go

```diff
@@ -311,7 +337,8 @@ func TestScheduler_runRecursiveJob(t *testing.T) {
 					taskArr = append(taskArr, num)
 				})
 				go func(tff TaskFunc) {
-					s.runRecursiveJob(context.Background(), tff, jobType)
+					sj := newSchedJob(context.Background(), s, "atype", tff)
```

Comment:
Need to pass correct jobType here for the limit semaphore to apply.

Reply:
🤦